docs/resources/idp_open_id_connect.md

```diff
@@ -15,20 +15,26 @@ If the external OpenID Connect identity provider returns a refresh token, it wil
 ```hcl
 resource "fusionauth_idp_open_id_connect" "OpenID" {
     application_configuration {
-        application_id = fusionauth_application.myapp.id
+        application_id      = fusionauth_application.myapp.id
         create_registration = true
-        enabled = true
+        enabled             = true
     }
-    button_text = "Login with OpenID Connect"
-    debug = false
-    name = "Super Awesome OpenID Connect Provider"
-    oauth2_authorization_endpoint = "https://acme.com/oauth2/authorization"
-    oauth2_client_id = "191c23dc-b772-4558-bd21-dc1cbf74ae21"
-    oauth2_client_secret ="SUsnoP0pWUYfXvWbSe5pvj8Di5nAxOvO"
+    oauth2_authorization_endpoint       = "https://acme.com/oauth2/authorization"
+    oauth2_client_id                    = "191c23dc-b772-4558-bd21-dc1cbf74ae21"
+    oauth2_client_secret                ="SUsnoP0pWUYfXvWbSe5pvj8Di5nAxOvO"
     oauth2_client_authentication_method = "client_secret_basic"
-    oauth2_scope = "openid offline_access"
-    oauth2_token_endpoint = "https://acme.com/oauth2/token"
-    oauth2_user_info_endpoint = "https://acme.com/oauth2/userinfo"
+    oauth2_scope                        = "openid offline_access"
+    oauth2_token_endpoint               = "https://acme.com/oauth2/token"
+    oauth2_user_info_endpoint           = "https://acme.com/oauth2/userinfo"
+    button_text                         = "Login with OpenID Connect"
+    debug                               = false
+    enabled                             = true
+    name                                = "Super Awesome OpenID Connect Provider"
```

Comment:
Not sure if this convention is followed elsewhere, but should we alphabetize these - which is to say, should `button_text`, `debug`, `enabled` and `name` come before `oauth2_*`?

Reply:
Great question!

A lot of the documentation is ordered based on the example JSON blob in the API documentation. Due to the manual copy+paste nature, it's been easier to order the examples based on the example payloads as you can run it through a JSON2HCL conversion (or manually eyeball it) and do the diff.

i.e. https://fusionauth.io/docs/v1/tech/apis/identity-providers/openid-connect has:

**_Example OpenID Connect Request JSON_**
```json
{
  "identityProvider" : {
    "applicationConfiguration" : {
      "1c212e59-0d0e-6b1a-ad48-f4f92793be32" : {
        "createRegistration" : true,
        "enabled" : true
      }
    },
    "oauth2": {
      "authorization_endpoint": "https://acme.com/oauth2/authorization",
      "client_id" : "191c23dc-b772-4558-bd21-dc1cbf74ae21",
      "client_secret" : "SUsnoP0pWUYfXvWbSe5pvj8Di5nAxOvO",
      "clientAuthenticationMethod": "client_secret_basic",
      "scope" : "openid offline_access",
      "token_endpoint" : "https://acme.com/oauth2/token",
      "userinfo_endpoint" : "https://acme.com/oauth2/userinfo"
    },
    "buttonText" : "Login with OpenID Connect",
    "debug": false,
    "enabled" : true,
    "name" : "Super Awesome OpenID Connect Provider",
    "tenantConfiguration": {
      "e872a880-b14f-6d62-c312-cb40f22af465": {
        "limitUserLinkCount": {
          "enabled": false,
          "maximumLinks": 42
        }
      }
    },
    "type" : "OpenIDConnect"
  }
}
```